pkg/constants/annotations.go

```diff
@@ -0,0 +1,17 @@
+package constants
```

Comment:
This is a decent sized anitpattern for Go. The convention is to define theise where they're used. Is there absolutely no possible way this convention can be upheld? Is this even necessary for this bug fix?

Reply:
It would be nice to have them in one place, or at least not define and export them from large modules that cannot be imported into other modules without creating import cycles, which usually results in yet another copy of the same const or var in a different package.

Upstream seems to be moving towards single easily importable locations for common annotations and labels, I would like for us to do the same.

Reply:
I let myself get side-tracked into moving all this around when I couldn't import `pkg/server` for 	`ControlPlaneRoleLabelKey` and noticed that we already have a couple other labels and annotations repeated within the codebase, but no this absolutely does not have to be fixed now,.

Reply:
I'm fine having them in one place but the package needs to be named appropriately, ie based on the contents and not the language construct.